common/persistence/nosql/nosqlplugin/cassandra/domain.go

```diff
@@ -575,3 +593,13 @@ func (db *cdb) deleteDomain(
 	query = db.session.Query(templateDeleteDomainQuery, ID).WithContext(ctx)
 	return db.executeWithConsistencyAll(query)
 }
+
+func getIsolationGroupFields(row *nosqlplugin.DomainRow) ([]byte, string) {
```

Comment:
is this method needed?

Reply:
It's a helper, it's used in two places. I don't feel strongly about it's use, but I don't see a problem with pulling out some safety logic into a more testable chunk?